sky/cli.py

```diff
@@ -1389,6 +1394,44 @@ def exec(
     sky.exec(task, backend=backend, cluster_name=cluster, detach_run=detach_run)
 
 
+def _get_in_progress_spot_jobs(
+        show_all_in_progress: bool = False) -> Tuple[Optional[int], str]:
+    """Get the in-progress spot jobs.
+
+    Returns:
+        A tuple of (num_in_progress_jobs, msg). If num_in_progress_jobs is None,
+        it means there is an error when querying the spot jobs. In this case,
+        msg contains the error message. Otherwise, msg contains the formatted
+        spot job table.
+    """
+    num_in_progress_jobs = None
+    try:
+        with ux_utils.suppress_output():
+            # Make the call slient
+            spot_jobs = core.spot_queue(refresh=False, skip_finished=True)
+            num_in_progress_jobs = len(spot_jobs)
+    except exceptions.ClusterNotUpError as e:
+        controller_status = e.cluster_status
+        if controller_status == global_user_state.ClusterStatus.INIT:
+            msg = ('Controller\'s latest status is detected to be INIT; jobs '
```

Comment:
```suggestion
            msg = ('Controller\'s latest status is INIT; jobs '
```
Perhaps it's ok to be more concise.